src/typescript/frontend/package.json

```
@@ -120,5 +120,5 @@
     "test:unit": "pnpm jest tests/unit",
     "vercel-install": "./submodule.sh && pnpm i"
   },
-  "version": "1.3.0"
+  "version": "1.3.1"
```

```suggestion
"version": "1.4.0"
```

(Since cult page styling is new?)

---

Honestly wasn't sure how to do the versioning since it's a frontend application haha, but you're probably right